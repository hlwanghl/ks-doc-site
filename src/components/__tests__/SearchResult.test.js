import React from 'react'
import renderer from 'react-test-renderer'
import SearchResult from '../SearchResult'

describe('SearchResult', () => {
  it('renders correctly', () => {
    const props = {
      results: []
    }

    const tree = renderer.create(<SearchResult {...props} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
