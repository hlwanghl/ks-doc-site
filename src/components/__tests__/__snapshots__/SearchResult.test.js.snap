// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResult renders correctly 1`] = `null`;
